zulip_bots/zulip_bots/bots/followup/followup.py

```diff
@@ -23,15 +23,24 @@ def usage(self) -> str:
             called "followup" that your API user can send to.
             '''
 
+    def initialize(self, bot_handler: Any) -> None:
+        self.config_info = bot_handler.get_config_info('followup', optional=False)
+        self.stream = self.config_info.get("stream", False)
+
     def handle_message(self, message: Dict[str, str], bot_handler: Any) -> None:
+        # Send help message
+        if message['content'] == 'help':
+            bot_handler.send_reply(message, self.usage())
+            return
+
         if message['content'] == '':
             bot_response = "Please specify the message you want to send to followup stream after @mention-bot"
             bot_handler.send_reply(message, bot_response)
         else:
             bot_response = self.get_bot_followup_response(message)
             bot_handler.send_message(dict(
                 type='stream',
-                to='followup',
+                to= 'followup',
```

Comment:
We don't put a space here.  I agree with you it sometimes looks nicer, but it's our convention.